doc/bootstrap.rst

```diff
@@ -6,22 +6,35 @@ A new Ceph cluster is deployed by bootstrapping a cluster on a single
 node, and then adding additional nodes and daemons via the CLI or GUI
 dashboard.
 
+The following example installs a basic three-node cluster. Each
+node will be identified by its prompt. For example, "[monitor 1]"
+identifies the first monitor, "[monitor 2]" identifies the second
```

Comment:
I'd switch 'monitor 1' etc to 'host 1'. (Some of the lines below already do this anyway.)

Reply:
Sage, if this PR goes through, I'll raise another one later that changes all instances of "monitor" to "host" where that's a smarter thing for the text to communicate.